spec/models/lesson_spec.rb

```diff
@@ -12,5 +12,16 @@
 
   describe 'associations' do
     it { is_expected.to belong_to(:course) }
+    it { is_expected.to have_many_attached(:files) }
+  end
+
+  describe 'owned by user' do
+    let(:user) { build(:user) }
+    let!(:course) { create :course, author: user }
+    let!(:lesson) { create :lesson, course: course }
+
+    it 'is owner' do
+      expect(lesson).to be_owner(user)
```

Comment:
и уже потом использовать вот так
```suggestion
      expect(lesson).to be_owner(course.author)
```

Reply:
Сделал